build-system/tasks/e2e/index.js

```diff
@@ -31,14 +32,16 @@ const WEBSERVER_TIMEOUT_RETRIES = 10;
 let webServerProcess_;
 
 function installPackages_() {
-  execOrDie('npx yarn --cwd build-system/tasks/e2e',
-      {'stdio': 'ignore'});
+  execOrDie('npx yarn --cwd build-system/tasks/e2e', {'stdio': 'ignore'});
+}
+
+function buildRuntime_() {
+  execOrDie('gulp build');
```

Comment:
We use `--fortesting` during `gulp dist` to make sure the production (or canary) config is applied to the AMP runtime. IIRC, `gulp build` automatically adds the configs without the `--fortesting` flag. If it turns out that this isn't true, we should make it so. This can be done in a separate PR.